ulimit.go

```diff
@@ -108,6 +109,17 @@ func ParseUlimit(val string) (*Ulimit, error) {
 	return &Ulimit{Name: parts[0], Soft: soft, Hard: *hard}, nil
 }
 
+// Verify that ulimit values work with current kernel
+func (u *Ulimit) Verify() error {
+	var rlimit syscall.Rlimit
```

Comment:
This won't compile on Windows (which would break the docker cli). Assuming we only need this on Linux/Unix-y systems (I guess just linux would do for now), could you move this to a `_linux.go` file (with the right build-tags), and create a stub implementation for other platforms? It could take the same approach as we did for the go-selinux package, and keep the exported function here, and a platform-specific, non-exported function in other files.